src/measure_theory/decomposition/lebesgue.lean

```diff
@@ -458,9 +458,6 @@ end lebesgue_decomposition
 
 open lebesgue_decomposition
 
-/-- **The Lebesgue decomposition theorem**: Any pair of finite measures `μ` and `ν`
-`have_lebesgue_decomposition`. That is to say, there exists a measure `ξ` and a measurable function
-`f`, such that `ξ` is mutually singular with respect to `ν` and `μ = ξ + ν.with_density f` -/
 @[priority 100] -- see Note [lower instance priority]
 instance have_lebesgue_decomposition_of_finite_measure
```

Comment:
Now that you have the general theorem for sigma-finite measures, you don't need this one to be an instance any more. Can you record it as a theorem, and use `local attribute [instance] have_lebesgue_decomposition_of_finite_measure` for the rest of the file? (And still give a docstring, with a pointer to the version for sigma-finite measures, explaining why this is not an instance)?